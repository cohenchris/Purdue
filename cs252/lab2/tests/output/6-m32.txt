0x577f6050
(nil)
[
	addr: 0080
	size: 4000
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x577f6098
(nil)
[
	addr: 0152
	size: 3928
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x577f60e0
(nil)
[
	addr: 0224
	size: 3856
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x577f6128
(nil)
[
	addr: 0296
	size: 3784
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
0x577f6008
0x577f6128
[
	addr: 0008
	size: 64
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0296
]

[
	addr: 0296
	size: 3784
	left_size: 64
	allocated: false
	prev: 0008
	next: NULLPTR
]

0x577f6008
0x577f6128
[
	addr: 0008
	size: 136
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0296
]

[
	addr: 0296
	size: 3784
	left_size: 64
	allocated: false
	prev: 0008
	next: NULLPTR
]

0x577f6008
0x577f6128
[
	addr: 0008
	size: 208
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0296
]

[
	addr: 0296
	size: 3784
	left_size: 64
	allocated: false
	prev: 0008
	next: NULLPTR
]

BREAK
0x577f6008
(nil)
[
	addr: 0008
	size: 4072
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
