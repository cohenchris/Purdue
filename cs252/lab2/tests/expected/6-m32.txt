
(nil)
[
	addr: 0080
	size: 4000
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]


(nil)
[
	addr: 0152
	size: 3928
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]


(nil)
[
	addr: 0224
	size: 3856
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]


(nil)
[
	addr: 0296
	size: 3784
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK


[
	addr: 0008
	size: 64
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0296
]

[
	addr: 0296
	size: 3784
	left_size: 64
	allocated: false
	prev: 0008
	next: NULLPTR
]



[
	addr: 0008
	size: 136
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0296
]

[
	addr: 0296
	size: 3784
	left_size: 64
	allocated: false
	prev: 0008
	next: NULLPTR
]



[
	addr: 0008
	size: 208
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: 0296
]

[
	addr: 0296
	size: 3784
	left_size: 64
	allocated: false
	prev: 0008
	next: NULLPTR
]

BREAK

(nil)
[
	addr: 0008
	size: 4072
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
